plugins/@grouparoo/bigquery/src/lib/table-import/getPropertyValue.ts

```diff
@@ -91,7 +91,7 @@ export const getPropertyValue: GetPropertyValueMethod = async ({
     }
   } catch (error) {
     throw new Error(
-      `Error with BigQuery SQL Statement: Query - \`${options}\`, Error - ${error}`
+      `Error with BigQuery SQL Statement: Query - \`${options.query}\`, Error - ${error}`
```

Comment:
I think we did want `options` here, maybe this should be `JSON.stringify(options)` - this way you can see the query and the ? params